src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarExtraLinks.tsx

```diff
@@ -55,7 +55,10 @@ ArtworkSidebarExtraLinksContainerProps
     type: Schema.Type.Link,
   }))
   onClickBuyNowFAQ() {
-    window.open(sd.APP_URL + "/buy-now-feature-faq", "_blank")
+    window.open(
+      "https://support.artsy.net/hc/en-us/sections/360008203114-Buy-Now-and-Make-Offer",
```

Comment:
Do we _always_ want to link to the `en-us` resource? I assume probably, given that we don't currently do i18n but I'm curious anyway.

Reply:
Good question. @wwiberg is there a plan to support multiple languages?

Reply:
not in the next few months/year but at some point I think it will make sense with Artsy growth